scripts/qa_crowbarsetup.sh

```diff
@@ -813,51 +814,51 @@ function onadmin_prepare_cloud_repos()
         add_bind_mount \
             "${localreposdir_target}/${CLOUDLOCALREPOS}/sle-11-x86_64/" \
             "${targetdir}"
-        echo $CLOUDLOCALREPOS > /etc/cloudversion
     else
         rsync_iso "$CLOUDSLE11DISTPATH" "$CLOUDSLE11DISTISO" "$targetdir"
-        cat "$targetdir/isoversion" > /etc/cloudversion
     fi
-    echo -n "This cloud was installed on `cat ~/cloud` from: " | \
-        cat - /etc/cloudversion >> /etc/motd
-    echo $cloudsource > /etc/cloudsource
 
     if [ ! -e "${targetdir}/media.1" ] ; then
         complain 35 "We do not have cloud install media in ${targetdir} - giving up"
     fi
 
-    zypper rr Cloud
-    safely zypper ar -f ${targetdir} Cloud
-
     if [ -n "$TESTHEAD" ] ; then
         case "$cloudsource" in
             GM4)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 ;;
             GM4+up)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 addcloud4testupdates
                 ;;
             GM5)
                 addcloud5pool
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 addsles12testupdates
                 ;;
             GM5+up)
                 addcloud5pool
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 addsles12testupdates
                 addcloud5testupdates
                 ;;
             develcloud4)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 ;;
             develcloud5|develcloud6)
-                addsp3testupdates
+                if [ -n "$want_sles12_admin" ]; then
+                  addsp3testupdates
+                else
+                  addsp3testupdates "sp3tup"
+                fi
                 addsles12testupdates
                 ;;
             susecloud6|M?|Beta*|RC*|GMC*|GM6|GM6+up)
-                addsp3testupdates
+                if [ -n "$want_sles12_admin" ]; then
+                  addsp3testupdates
```

Comment:
Because for now, we still setup the SLE11 repos in /srv/tftpboot. This is indeed something we can drop at some point; just didn't want to drop part of it and not everything.